port/unix/omrsysinfo.c

```diff
@@ -236,6 +241,44 @@ struct  {
 #endif
 };
 
+#if defined(LINUX)
+
+#define OMR_CGROUP_V1_MOUNT_POINT "/sys/fs/cgroup"
+
+/* Currently 12 subsystems/resource controllers are defined.
+ */
+typedef enum CgroupSubsystems {
```

Comment:
I missed this one in the first review. Sorry.  If this enum ever "escapes" this file, it should probably be prefixed with "OMR".